### PR TITLE
Milestone 2 - Chloe Lo

### PR DESCRIPTION
**Pull request by sylochloe**
# Milestone: 2
# Developer: Chloe Lo

### Pull Request Summary
This is a new pull request for turning in the Milestone 1 for review by the Directors. Thank you.

### Pull Request Checklist
- [x] Code is neat, readable, and works
- [x] Comments are appropriate
- [x] The commit message follows our guidelines
- [x] The milestone number is specified
- [x] The developer name is specified
- [x] The summary is completed

### Reviewer Checklist  - PULL REQUEST REVIEWER ONLY

IMPORTANT: The rest of the sections in this checklist should only be filled out by authorized pull request reviewers. If you are the individual template contributor, do not fill out the rest of the fields or check the boxes.

NOTE: Milestones can only be completed when all boxes are checked.

- [x] The code is fully reviewed
- [x] Meaningful feedback is given
- [x] Comment that you have reviewed the code
- [x] This box is checked

### CODE REVIEW

File: milestone2.ts [hunk 1]
```diff
@@ -4,75 +4,75 @@ interface Volunteer {
   name: string;
   age: number;
   email: string;
-  position: "volunteer" | "staff";
+  position: 'volunteer' | 'staff';
```
Comment (vwinstea) [inline milestone2.ts:7]: Love to see some autoformatting :)

**Review by vwinstea — COMMENTED** [review]
Looks great!

**Comment by vwinstea** [comment]
Milestone 2 complete!

**Comment by sylochloe** [comment]
Thank you.

**Pull request closed without merging**